charts/pega/templates/pega-tier-deployment.yaml

```diff
@@ -32,7 +32,7 @@
 {{ end }}
 {{ end }}
 
-{{ if (eq (include "performUpgradeAndDeployment" $) "true") }}
+{{ if ( and  (eq (include "performUpgradeAndDeployment" $) "true") (eq $.Values.installer.upgrade.upgradeType "zero-downtime")) }}
```

Comment:
Other than zero-downtime upgrade , when can we have 'performUpgradeAndDeployment' as true.
If we only keep 2nd condition , will it meet the objective here?

Reply:
`performUpgradeAndDeployment` can now be set for both ZDT and Custom upgrades. In case of failure, the user can switch to a Custom upgrade. Therefore, we do not require any `waitForPegaDBZDTUpgrade` pods, which would block Custom upgrades. Hence, we are restricting this to upgradeType  `zero-downtime` and `performUpgradeAndDeployment` to true